erpnext/www/authorize_document.html

```diff
@@ -0,0 +1,73 @@
+{% extends "templates/web.html" %}
+
+{% block title %} Authorize Document {% endblock %}
+{% block style %}
+	<link type="text/css" rel="stylesheet" href="/assets/bloomstack_core/css/authorize_document.css">
```

Comment:
path to css is not proper, connect with Vishal for the proper path

Reply:
Hi Sahil,
The PR has been updated with suggested changes.

PR link: https://github.com/Bloomstack/erpnext/pull/1659